backend/config/application.rb

```diff
@@ -11,7 +11,7 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 7.0
 
-    Rails.application.routes.default_url_options[:host] = ENV.fetch("BACKEND_URL", "http://localhost:4000")
+    Rails.application.routes.default_url_options[:host] = URI.parse(ENV.fetch("BACKEND_URL", "http://localhost:4000")).host
```

Comment:
I don't know why you changed that. It was actually ok, to have host with port, now it is incorrect in api-docs (without port). I would drop "/backend" path in BACKEND_URL.

Reply:
Hmm, forgot about port at swagger examples, but BACKEND_URL kind of implies that it is full path to place where Rails is run so it made more sense for me to keep /backend there and fix host. Whatever, I dont mind to change it back, but maybe it would be worthy to rename BACKEND_URL to BACKEND_HOST

Reply:
Or it can be defined same way as MAILER_DEFAULT_HOST:
config.action_mailer.default_url_options = {host: ENV["MAILER_DEFAULT_HOST"], protocol: "https"}
at env files. This way, we can have port and protocol, but still keep /backend path